### PR TITLE
Fix fromHTML stripping root tags

### LINKED ISSUE #33: fromHTML strips root tags
The use of a \<div> for parsing HTML with the browser causes the root tags (\<html>, \<body>, \<head>) to be removed from the returned document.

### PR DESCRIPTION
**Pull request by wtfaremyinitials**
Fixes #33

### CODE REVIEW

**Comment by wtfaremyinitials** [comment]
@marcelklehr?

File: index.js [hunk 1]
```diff
@@ -35,10 +35,26 @@ function createVNode(domNode, key) {
...
+  try {
+    // Everything except iOS 7 Safari, IE 8/9, Andriod Browser 4.1/4.3
+    var parser = new DOMParser();
+    var doc = parser.parseFromString(html, 'text/html');
+    rootNode = doc.documentElement;
+  } catch(e) {
+    // Old browsers
+    var ifr = document.createElement('iframe');
+    ifr.setAttribute('data-content', html);
+    ifr.src = 'javascript: window.frameElement.getAttribute("data-content");';
```
Comment (marcelklehr) [inline index.js]: Shouldn't this do more, like insert the actual html?

**Comment by marcelklehr** [comment]
Sorry, I forgot about this...

File: index.js (hunk 1 above)
Comment (wtfaremyinitials) [inline index.js]: @marcelklehr The HTML is inserted using a little trick to get old browsers working. The URL of the page in the `<iframe>` is set to `javascript: window.frameElement.getAttribute("data-content");`. This tells the browser to set the page's source to be equal to the frameElement's attribute `data-content`, which contains the HTML to be parsed.

Comment (wtfaremyinitials) [inline index.js]: A reference implementation of this trick can be found at [jugglinmike/srcdoc-polyfill](https://github.com/jugglinmike/srcdoc-polyfill). The README explains it quite well.

Comment (marcelklehr) [inline index.js]: aah, I get it. Will test and merge ASAP

**Comment by marcelklehr** [comment]
Soo, finally. Thank you for this pr!

**Comment by wtfaremyinitials** [comment]
@marcelklehr no problem! thanks for the merge!

**Pull request merged**